### DISCUSSION TITLE
Email rate limit exceeded.

### DISCUSSION THREAD

**Discussion by Axiom0599** [supabase/supabase#15796] [Questions] [answered]
When I submit my form I get an error that says 'Email rate limit exceeded'.

**Comment by GaryAustin1** [comment]
The built in SMTP service has limits...  You should use your own SMTP service to overcome those.
https://supabase.com/docs/guides/platform/going-into-prod#auth-rate-limits

**Reply by GaryAustin1** [reply, marked as answer]
Detailed explanation.
https://github.com/orgs/supabase/discussions/15896

**Comment by Xananax** [comment]
I have raised the limit to 200, I use a custom SMTP, but still get 

```
 AuthApiError: Rate limit exceeded
    at (node_modules/@supabase/gotrue-js/dist/module/lib/fetch.js:31:10)
    at (node_modules/@supabase/gotrue-js/dist/module/lib/fetch.js:68:8)
    at (node_modules/@supabase/gotrue-js/dist/module/lib/fetch.js:53:17)
    at (node_modules/@supabase/gotrue-js/dist/module/GoTrueClient.js:1191:23)
    at (node_modules/@supabase/gotrue-js/dist/module/lib/helpers.js:190:35) {
  __isAuthError: true,
  name: 'AuthApiError',
  status: 429
}
```
in my logs.

My registration require no confirmation, so supposedly, the only emails I need to send are OTP (a side feature, we invite people to use email + pass primarily) or password reset tokens.

Am I having 200 OTPs per hour? How can I check this? The supabase logs show no errors and I am not managing to find what the rate limit tracks exactly or how to see if we're close.
